rabbitmq-server-3.13.yaml

```diff
@@ -35,14 +35,10 @@ environment:
 pipeline:
   - uses: fetch
     with:
-      expected-sha256: c0f15799f7e3885c301919ae0ecb87e2bce84a151003302dd27e9e6342d78284
+      expected-sha256: cbaac76690215d5b8eeac9f5f1946f6c6f2a6a34c255d13b896927f8a5d48702
       uri: https://github.com/rabbitmq/rabbitmq-server/releases/download/v${{package.version}}/rabbitmq-server-${{package.version}}.tar.xz
 
-  - runs: |
-      cd deps/rabbitmq_cli
-      mix format
-
-  - runs: make
+  - uses: autoconf/make
```

Comment:
What prompted this change? Can you add context in the commit message?

Reply:
I am not sure why `mix format` was there cause it should automatically run via make command.

the recommended way of running make file is to use automake/make pipeline. previously It was using the same but there was an [error](https://github.com/wolfi-dev/os/pull/16016#issuecomment-2079041660). and as now the error is gone I move it back to use the pipeline.